jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/CachedContentFactory.java

```diff
@@ -381,7 +382,7 @@ public class CachedHttpContent implements HttpContent
         private final MimeTypes.Type _mimeType;
         private final HttpField _contentLength;
         private final HttpField _lastModified;
-        private final long _lastModifiedValue;
+        private final Instant _lastModifiedValue;
```

Comment:
This is good, but I think that while we're at it, we should go one step further and make also `_lastAccessed` below an `Instant`.

With this change, we need also to fix the `Comparator` in `shrinkCache()` which is broken as it does not test `_contentLengthValue` properly (only when `<`, but should do also the `>` branch).

Reply:
Done.